.vscode/extensions.json

```diff
@@ -1,5 +1,5 @@
 {
   // See http://go.microsoft.com/fwlink/?LinkId=827846
   // for the documentation about the extensions.json format
-  "recommendations": ["dbaeumer.vscode-eslint", "amodio.tsl-problem-matcher", "esbenp.prettier-vscode", "ms-python.python"]
+  "recommendations": ["ms-python.python", "swellaby.workspace-config-plus"]
```

Comment:
Perhaps there are more (like `atopile`), but I don't think the current ones were it

Reply:
Me too, re. the popularity thing. I'm also hesitant to not make the developer experience good though, and with that, good settings off the bat etc..., so I'm okay with the exception for the workspace-config-plus specifically to do this